src/gulpfile.js

```diff
@@ -255,6 +255,25 @@ gulp.task('build-html', ['add-revision-numbers'], function () {
   .pipe(gulp.dest(paths.build));
 });
 
+gulp.task('build-mseditor', ['template-editor'], function () {
+  return gulp.src([
+    paths.app + '/template-editor/index.html'
+  ])
+  .pipe(gulpInject(gulp.src(paths.app + '/env/template-editor/' + process.env.NODE_ENV + '.js'), {
+    transform: function(filePath, file) {
+      return '<script>' + file.contents.toString() + '</script>';
```

Comment:
As I understand, it is injecting a module as inline script.

What do you think about bundle all .js files in a single one in place of doing it?

Reply:
I will check it, we can add the serviceAdapter.js and the environment js  file defining the backend configuration.